ci_microshift.sh

```diff
@@ -5,7 +5,8 @@ set -exuo pipefail
 sudo yum install -y make golang
 
 ./shellcheck.sh
-MICROSHIFT_PRERELEASE=yes ./microshift.sh
```

Comment:
This line was changed in the first commit in this series, should they be squashed together?

Reply:
These are different commit the `MICROSHIFT_PRERELEASE` suppose to get the rpm packages from the mirror with `ocp-dev-preview` but since the mirror location is changed with `ocp` when they start rolling the rc bits so meanwhile we also need to change the `MIRROR` link.

Reply:
Shouldn't the code handling `MICROSHIFT_PRERELEASE` be reworked then? Looking at microshift.sh, this is also not correct as there is a `release-4.15` branch in microshift repo
```
    if [ -n "${MICROSHIFT_PRERELEASE-}" ]; then
       git clone -b main https://github.com/openshift/microshift.git
    else
       git clone -b release-4.14 https://github.com/openshift/microshift.git
    fi
```

Reply:
Right, I created https://github.com/crc-org/snc/pull/844 one and will rebase accordingly.